src/cmd/manage_nsfs.js

```diff
@@ -259,7 +259,7 @@ async function delete_bucket(data, force) {
         }
         await native_fs_utils.folder_delete(bucket_temp_dir_path, fs_context_fs_backend, true);
         await config_fs.delete_bucket_config_file(data.name);
-        return { code: ManageCLIResponse.BucketDeleted, detail: '', event_arg: { bucket: data.name } };
+        return { code: ManageCLIResponse.BucketDeleted, detail: `Bucket ${data.name} has been successfully deleted.`, event_arg: { bucket: data.name } };
```

Comment:
You can probably add a small line (as I assume we test the bucket deletion) to test the change in file: test_nc_nsfs_bucket_cli.test.js

Reply:
Sure.